api/lib/schema.js

```diff
@@ -386,11 +375,11 @@ const typeDefs = gql`
   }
 
   type TransactionMetadata {
-    gasEstimate: Int!
+    gasEstimate: Int
     gasPrices: [GasPrice]
     chainAppliedFees: ChainAppliedFees
-    accountSequence: Int
-    accountNumber: Int
+    accountSequence: String
```

Comment:
why strings?

Reply:
because Int fails. You need to stringify it for the message. At least, I tried it and it failed

Reply:
why does int fail and where?

Reply:
give me a sec

Reply:
Yes, it fails when making and this is the error message:

<img width="440px" src="https://user-images.githubusercontent.com/40721795/88321591-befaa900-cd1f-11ea-883c-cf0157482e8c.png">

Reply:
well you would need to convert it when creating the signature...

Reply:
Can't figure out the change myself. WIll merge